src/pypromice/postprocess/wmo_config.py

```diff
@@ -17,8 +17,8 @@
     'discontinued': ['CEN1','TAS_U','QAS_A','NUK_N','THU_U','JAR','SWC'],
     'no_instantaneous': ['ZAK_L','ZAK_U','KAN_B'], # currently not transmitting instantaneous values
     'suspect_data': [], # instantaneous data is suspect
-    'use_v3': ['KPC_L','NUK_U','ZAK_U','QAS_U'], # use v3 versions instead (but registered IDs are non-v3 names)
-    'v3_bad': ['KPC_Uv3','QAS_Lv3'] # KPC_Uv3 years are 2056, QAS_Lv3 new ablation sensor w/different txt fields?
+    'use_v3': ['KPC_L','KPC_U','NUK_U','ZAK_U','QAS_U'], # use v3 versions instead (but registered IDs are non-v3 names)
```

Comment:
Did we have a resolution on @BaptisteVandecrux comment about the latency of KPC_Uv3 data? He said that it may be +/- 15 mins out from the actual timestamp right?